streamz/tests/test_dask.py

```diff
@@ -131,6 +131,67 @@ def test_buffer(c, s, a, b):
     assert source.loop == c.loop
 
 
+@pytest.mark.slow
+def test_filter(backend):
```

Comment:
You need to remove the `backend` arg here, and in the `scatter` statement. Streamz only has a dask backend at the moment.

Reply:
Should I remove the backend for all the four tests?

Reply:
Should I remove the backend for all the four tests I have added?

Reply:
yes please

Reply:
Already changed the four tests but the build is still failed